src/libraries/LPFeeLibrary.sol

```diff
@@ -9,24 +9,51 @@ library LPFeeLibrary {
     /// @notice Thrown when the static or dynamic fee on a pool exceeds 100%.
     error FeeTooLarge();
 
-    uint24 public constant STATIC_FEE_MASK = 0x7FFFFF;
+    uint24 public constant FEE_MASK = 0x7FFFFF;
+    uint24 public constant OVERRIDE_MASK = 0xBFFFFF;
+
+    // the top bit of the fee is used to signal if a Pool's is dynamic
     uint24 public constant DYNAMIC_FEE_FLAG = 0x800000;
 
+    // the second bit of the fee (returned by beforeSwap) is used to signal if the stored fee should be overridden
```

Comment:
```suggestion
    // the second bit of the fee returned by beforeSwap is used to signal if the stored LP fee should be overridden in this swap
```

Reply:
maybe then 1 more comment on a line below saying only dynamic-fee pools can return a fee from the beforeSwap hook